subsys/net/l2/openthread/Kconfig

```diff
@@ -257,6 +257,19 @@ config OPENTHREAD_SLAAC
 	help
 	  Enable SLAAC capability in OpenThread stack
 
+config OPENTHREAD_STACK_VENDOR_OUI
+	int "Vendor Organizationally Unique Identifier"
+	default 16043574
+	help
+	  Custom vendor OUI for OpenThread,
+	  which makes up 24 bits of MAC address
+
+config OPENTHREAD_EUI64_CUSTOM_SOURCE
+	bool "Custom EUI64 source support"
+	help
+	  Enable providing EUI64 address
```

Comment:
I'd make it explicit in the help message, that `otPlatRadioGetIeeeEui64` has to be implemented in that case.

Reply:
Done